test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -307,6 +307,387 @@ public class PeekDefinitionTests
+        /// Test get definition for a scalar valued function object that doesn't exist with active connection. Expect null locations
+        /// </summary>
+        [Fact]
+        public async Task GetScalarValuedFunctionDefinitionWithNonExistantFailureTest()
+        {
+            // Get live connectionInfo and serverConnection
+            ConnectionInfo connInfo = await TestObjects.InitLiveConnectionInfoForDefinition();
+            ServerConnection serverConnection = TestObjects.InitLiveServerConnectionForDefinition(connInfo);
+
+            PeekDefinition peekDefinition = new PeekDefinition(serverConnection, connInfo);
+            string objectName = "doesNotExist";
+            string schemaName = "dbo";
+            string objectType = "FUNCTION";
+
+            Location[] locations = peekDefinition.GetSqlObjectDefinition(peekDefinition.GetScalarValuedFunctionScripts, objectName, schemaName, objectType);
+            Assert.Null(locations);
+        }
+
+        /// <summary>
+        /// Test get definition for a table valued function object that doesn't exist with active connection. Expect null locations
+        /// </summary>
+        [Fact]
+        public async Task GetTableValuedFunctionDefinitionWithNonExistantObjectFailureTest()
+        {
+            // Get live connectionInfo and serverConnection
+            ConnectionInfo connInfo = await TestObjects.InitLiveConnectionInfoForDefinition();
+            ServerConnection serverConnection = TestObjects.InitLiveServerConnectionForDefinition(connInfo);
+
+            PeekDefinition peekDefinition = new PeekDefinition(serverConnection, connInfo);
+            string objectName = "doesNotExist";
+            string schemaName = "dbo";
+            string objectType = "FUNCTION";
+
+            Location[] locations = peekDefinition.GetSqlObjectDefinition(peekDefinition.GetTableValuedFunctionScripts, objectName, schemaName, objectType);
+            Assert.Null(locations);
+        }
+
+        /// <summary>
+        /// Test get definition for a scalar valued function object with active connection. Expect non-null locations
+        /// </summary>
+        [Fact]
+        public async Task GetScalarValuedFunctionDefinitionWithoutSchemaNameSuccessTest()
+        {
+            // Get live connectionInfo and serverConnection
+            ConnectionInfo connInfo = await TestObjects.InitLiveConnectionInfoForDefinition();
+            ServerConnection serverConnection = TestObjects.InitLiveServerConnectionForDefinition(connInfo);
+
+            PeekDefinition peekDefinition = new PeekDefinition(serverConnection, connInfo);
+            string objectName = "addTwo";
+            string schemaName = null;
+            string objectType = "FUNCTION";
+
+            Location[] locations = peekDefinition.GetSqlObjectDefinition(peekDefinition.GetScalarValuedFunctionScripts, objectName, schemaName, objectType);
+            Assert.NotNull(locations);
+            Cleanup(locations);
```

Comment:
Why do you cleanup here and not other places? Should cleanup be done for all tests?